groups/instance/lb.tf

```diff
@@ -0,0 +1,103 @@
+resource "aws_lb" "artifactory" {
+  name               = "${var.environment}-${var.service}-lb"
+  internal           = true
+  load_balancer_type = "application"
+  security_groups    = [aws_security_group.alb_security_group.id]
+  subnets            = local.automation_subnet_ids
+
+  enable_deletion_protection = true
+
+  tags = {
+    Name        = "${var.environment}-${var.service}-lb"
+  }
+}
+
+resource "aws_lb_target_group" "front_end_8081" {
+  name     = "${var.environment}-${var.service}-tg-8081"
+  port     = 8081
+  protocol = "HTTP"
+  vpc_id   = data.aws_vpc.placement.id
+
+  health_check {
+    enabled             = true
+    interval            = 30
+    path                = "/"
+    port                = "8081"
+    timeout             = 3
+    healthy_threshold   = 3
+    unhealthy_threshold = 3
+    protocol            = "HTTP"
+    matcher             = "200"
+  }
+}
```

Comment:
This target group isn't required as we won't be passing traffic through to the instance on port 8081